bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx93-v3-en.html

```diff
@@ -0,0 +1,62 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}wnp93-v3{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_93') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
```

Comment:
Because this version doesn't support dark-mode, the Firefox wordmark logo disappears when I have the preference set: 

![image](https://user-images.githubusercontent.com/400117/135418194-cf09de40-b0f0-447a-b3ec-201b27678b5d.png)